src/vivarium_csu_alzheimers/constants/metadata.py

```diff
@@ -40,6 +40,15 @@
 ARTIFACT_COLUMNS = pd.Index([f"draw_{i}" for i in range(DRAW_COUNT)])
 
 
+FORECAST_NC_FNAME_DICT = {
```

Comment:
In the constants package, there is a paths.py so you should save the paths there and then import them here into this dict. You could also save the whole dict there if you wanted to.

Nit: try not to use abbreviations unless really necessary. You could do something like FORECAST_DATA_FILEPATHS_DICT.